src/rust/Cargo.toml

```diff
@@ -52,11 +52,11 @@ optional = true
 version = "0.0.17"
 git = "https://github.com/stellar/rs-soroban-env"
 package = "soroban-env-host"
-rev = "c2e1c21cf8d44db23a159090e3cbaab741860295"
+rev = "f5c6c0f42b4094481b7f32ecac825ce27c7114e0"
```

Comment:
Why not `5dc7547a4b15028787d57a849a7684802d899fc7` like the rest?

Reply:
This is supposed to point to the "previous" host version (the comment has an explanation).

Reply:
ah, sorry